docs/advanced/exceptions.rst

```diff
@@ -43,15 +47,23 @@ at its exception handler.
 |                                      | of bounds access in ``__getitem__``, |
 |                                      | ``__setitem__``, etc.)               |
 +--------------------------------------+--------------------------------------+
-| :class:`pybind11::value_error`       | ``ValueError`` (used to indicate     |
-|                                      | wrong value passed in                |
-|                                      | ``container.remove(...)``)           |
-+--------------------------------------+--------------------------------------+
 | :class:`pybind11::key_error`         | ``KeyError`` (used to indicate out   |
 |                                      | of bounds access in ``__getitem__``, |
 |                                      | ``__setitem__`` in dict-like         |
 |                                      | objects, etc.)                       |
 +--------------------------------------+--------------------------------------+
+| :class:`pybind11::value_error`       | ``ValueError`` (used to indicate     |
+|                                      | wrong value passed in                |
+|                                      | ``container.remove(...)``)           |
```

Comment:
Why was this moved? This adds to the diff (and future `blame`) without actually being a difference.

Reply:
I did this because it's the same order as in the code now, so it ease the work for maintainers, as they can easily check everything is correct, as it's in the same order etc.

The work is really simple here: open the code, copy/paste the names of the exceptions into this table^^

Reply:
OK, that's a good reason indeed. That just wasn't immediately clear :-)